packages/subiquity_client/lib/src/types.dart

```diff
@@ -5,6 +5,36 @@ part 'types.g.dart';
 
 // ignore_for_file: invalid_annotation_target
 
+// subiquity serializes `Dict[int, Any]` as `List[List[int, Any]]`
+class JsonMapConverter
+    implements JsonConverter<Map<int, String>, List<dynamic>> {
+  const JsonMapConverter();
+
+  @override
+  Map<int, String> fromJson(List<dynamic> json) {
+    return {
+      for (final entry in json) entry[0] as int: entry[1] as String,
+    };
+  }
+
+  @override
+  List<dynamic> toJson(Map<int, String> values) {
+    return [
+      for (final entry in values.entries) [entry.key, entry.value],
+    ];
+  }
+}
+
+// subiquitycore.models.network
+class NetDevInfo {
+  const NetDevInfo._();
+  factory NetDevInfo.fromJson(Map<String, dynamic> json) => NetDevInfo._();
+  Map<String, dynamic> toJson() => {};
+}
+
+// Union[Partition, Gap]
+typedef DiskObject = PartitionOrGap;
```

Comment:
the generator doesn't know that `Union[Partition, Gap] == DiskObject` so this is for backward compatibility